script/foundry/utils/BroadcastManager.s.sol

```diff
@@ -13,17 +13,17 @@ contract BroadcastManager is Script {
         uint256 deployerPrivateKey;
         if (block.chainid == 1) { // Tenderly mainnet fork
             deployerPrivateKey = vm.envUint("MAINNET_PRIVATEKEY");
-            deployer = vm.envAddress("MAINNET_DEPLOYER_ADDRESS");
+            deployer = vm.addr(deployerPrivateKey);
             multisig = vm.envAddress("MAINNET_MULTISIG_ADDRESS");
             vm.startBroadcast(deployerPrivateKey);
         } else if (block.chainid == 11155111) {
             deployerPrivateKey = vm.envUint("SEPOLIA_PRIVATEKEY");
-            deployer = vm.envAddress("SEPOLIA_DEPLOYER_ADDRESS");
+            deployer = vm.addr(deployerPrivateKey);
             multisig = vm.envAddress("SEPOLIA_MULTISIG_ADDRESS");
             vm.startBroadcast(deployerPrivateKey);
         } else if (block.chainid == 31337) {
-            multisig = address(0x456);
-            deployer = address(0x999);
+            require(deployer != address(0), "Deployer not set");
```

Comment:
It might be not addressed in this PR. Should it add support of story-network.

Reply:
Yes, will open a PR to add this separately, but thanks for the heads up.